### PR TITLE
Use Aroma for modding vWii

### PR DESCRIPTION
**Pull request by VannyBuns**
This will fix the issue of still needing Tiramisu to mod the vWii.

### CODE REVIEW

File: docs/user-guide/vwii/installing-homebrewchannel.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 ## Installing the Homebrew Channel
 
-1. Boot into [Tiramisu](browser-exploit).
-1. Launch the Homebrew Launcher, by opening the Mii Maker.
-1. Launch the vWii Compat Installer.
+1. Boot into [Aroma](browser-exploit).
+1. Launch the `vWii Compat Installer.rpx` on the Wii U Menu.
```
Comment (Lazr1026) [inline docs/user-guide/vwii/installing-homebrewchannel.md:6]:
```suggestion
1. Launch the `vWii Compat Installer` from the Wii U Menu.
```
The `.wuhb` will take priority over the `.rpx`.

File: docs/user-guide/vwii/sd-preparation.md [hunk 2]
```diff
@@ -10,22 +10,23 @@ We will now start by placing the required Homebrew files on the SD Card.
...
-    
+
+?> Unlike Wii U homebrew, vWii apps go in a separate apps folder on the root. Compat Installer is the **only** Wii U app here. 
 
 ### What You Need
 
-- The latest files from [Tiramisu for your café](https://tiramisu.foryour.cafe).
-    - Click on `Download Tiramisu`.
-    - If you have already installed Tiramisu, you do not need to redownload it.
+- The latest files from [Aroma for your café](https://aroma.foryour.cafe).
```
Comment (Lazr1026) [inline docs/user-guide/vwii/sd-preparation.md:18]: Should probably have a note for the checkboxes as well.

File: docs/user-guide/vwii/sd-preparation.md [hunk 3]
```diff
@@ -10,22 +10,23 @@ We will now start by placing the required Homebrew files on the SD Card.
...
+    - Click on `Download Payloads`and `Download Base Aroma`.
+    - If you have already installed Aroma, you do not need to redownload it.
 - The latest release of the [Compat Title Installer](https://hb-app.store/wiiu/CompatTitleInstaller).
-- The <a href="docs/files/Patched_IOS80_Installer_for_vWii.zip" download>Patched IOS 80 Installer for vWii</a>. ([Source](https://github.com/Lazr1026/Patched-IOS80-Installer-for-vWii))
-- The <a href ="docs/files/d2x_cIOS_Installer.zip" download>d2x cIOS Installer</a>.
+- The [Patched IOS 80 Installer for vWii.](https://wiiu.hacks.guide/docs/files/Patched_IOS80_Installer_for_vWii.zip) ([Source](https://github.com/Lazr1026/Patched-IOS80-Installer-for-vWii))
+- The [d2x_cIOS_Installer.zip](https://wiiu.hacks.guide/docs/files/d2x_cIOS_Installer.zip).
```
Comment (Lazr1026) [inline docs/user-guide/vwii/sd-preparation.md:23]: Does linking this way actually work?

File: docs/user-guide/vwii/sd-preparation.md [hunk 4]
```diff
@@ -10,22 +10,23 @@ We will now start by placing the required Homebrew files on the SD Card.
...
 
 ## Instructions
 
 1. Insert your Wii U's SD Card into your PC.
 1. Copy the `apps` folder from the `Patched_IOS80_Installer_for_vWii.zip` file to the root of your SD Card.
 1. Copy the `d2x-cios-installer` from the `d2x_cIOS_Installer.zip` file to the apps folder on the root of your SD Card.
-1. Copy the contents of the downloaded Tiramisu *`.zip`* file to the root of your SD Card.
+1. Copy the contents of the downloaded Base Aroma and payloads `.zip` file to the root of your SD Card.
```
Comment (Lazr1026) [inline docs/user-guide/vwii/sd-preparation.md:30]:
```suggestion
1. Copy the contents of the downloaded Base Aroma and payloads `.zip` files to the root of your SD Card.
```

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

File: docs/user-guide/vwii/installing-homebrewchannel.md (hunk 1 above)
Comment (VannyBuns) [reply docs/user-guide/vwii/installing-homebrewchannel.md:6]: Don't know how I forgot about the WUHB. But indeed

File: docs/user-guide/vwii/sd-preparation.md (hunk 3 above)
Comment (VannyBuns) [reply docs/user-guide/vwii/sd-preparation.md:23]: Tested it myself

File: docs/user-guide/vwii/sd-preparation.md (hunk 2 above)
Comment (VannyBuns) [reply docs/user-guide/vwii/sd-preparation.md:18]: Would it be better as a bullet point or an info box

Comment (Lazr1026) [reply docs/user-guide/vwii/sd-preparation.md:18]: bullet, like the main guide.

Comment (VannyBuns) [reply docs/user-guide/vwii/sd-preparation.md:18]: Sounds good

**Pull request merged**